LuaMenu/widgets/api_user_handler.lua

```diff
@@ -138,8 +139,10 @@ local function GetUserSyncStatus(userName, userControl)
 	end
 	if userBattleInfo.sync == 1 then
 		return IMAGE_READY
-	else
+	elseif userBattleInfo.sync == 2 then
```

Comment:
Wouldn't this break ZK? Do they use 2 to represent unsynced?

Reply:
Googlefrog says ZK uses :
public enum SyncStatuses
    {
        Unknown = 0,
        Synced = 1,
        Unsynced = 2
    }